### ISSUE TITLE
 CopilotForXcode这几天突然无法使用了，大家有这样的情况吗？

### ISSUE THREAD

**Issue by juefeiyechen** [github/CopilotForXcode#69]
**Describe the bug**

**Versions**
- Copilot for Xcode: [e.g. 0.25.0]
- Xcode: [e.g. 16.0]
- macOS: [e.g. 14.6.1]

 CopilotForXcode这几天突然无法使用了，大家有这样的情况吗？

**Comment by testforstephen** [comment]
你能详细说明一下遇到的问题吗？谢谢！
Can you share more details with the issue you have got? thanks.

**Comment by dushihao** [comment]
俺也一样，很奇怪，配置都没什么问题🤔

**Comment by HisongMo** [comment]
> 俺也一样，很奇怪，配置都没什么问题🤔
> 
> 问题就是代码补全不生效了

是不是用量到上限了

**Comment by ildream** [comment]
> > 俺也一样，很奇怪，配置都没什么问题🤔
> > 问题就是代码补全不生效了
> 
> 是不是用量到上限了

你的意思是免费版有2000次code completions的限制？

**Comment by shixiaoda** [comment]
我都付费了，也不能用了...

**Comment by dushihao** [comment]
> > 俺也一样，很奇怪，配置都没什么问题🤔
> > 问题就是代码补全不生效了
> 
> 是不是用量到上限了

好像是的，哈哈哈

**Comment by DeMartine** [comment]
Yeah, I've noticed that too! CopilotForXcode has been acting up lately and just doesn’t seem to work like it used to. It’s frustrating when tools we rely on suddenly stop functioning. Have you tried restarting Xcode or checking for any updates? Sometimes that can help. Let’s see if anyone else has found a solution!
